src/drawing.rs

```diff
@@ -425,22 +426,25 @@ impl Animator {
             self.frame_index = (self.frame_index + 1) % self.animation.len();
         }
 
-        current_frame.text.clone()
+        current_frame.to_owned()
```

Comment:
I think there is no need to clone it if we return a reference of `AnimationFrame` from this method